ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -3,6 +3,7 @@
    [camel-snake-kebab.core :as csk]
    [cheshire.core :as json]
    [clojure.java.io :as io]
+   [clojure.java.jdbc :as jdbc]
```

Comment:
Our services namespaces are not meant to communicate with the database or external APIs. Those should be in a `data` namespace.